easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,62 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/03
+easyblock = 'Binary'
+# Use generic easyblock in order to adapt to variable naming scheme
```

Comment:
what's wrong with the MotionCor2 easyblock exactly? if there are issues with naming schemes, can't we fix them in the easyblock?